### PR TITLE
Use rich text (tiptap) for measures in place

### PR DESCRIPTION
**Pull request by reinhardt**
Refs syslabcom/scrum#596

See also #473 for styles and #437 for regular (not in-place) measures

### CODE REVIEW

**Comment by reinhardt** [comment]
Note to self: Check with vanilla euphorie and osha oira

File: src/euphorie/client/browser/templates/webhelpers.pt [hunk 1]
```diff
@@ -213,13 +213,24 @@
...
+                          name="measure-custom-${measure/solution_id}"
+                          placeholder="${view/placeholder_add_extra}"
+                          rows="3"
+                          data-pat-tiptap="
+                            toolbar-external: #assessment-toolbar;
+                            source-panel: #pat-modal.panel-html-source-editor;
+                            image-panel: #pat-modal.image-picker;
+                            link-panel: #pat-modal .link-panel;
+                            context-menu-link: #tiptap-context-menu-hyperlink-fieldname-comments
+                            url-scheme-mentions: https://quaive.cornelis.amsterdam/users/{USER};
```
Comment (ale-rt) [inline src/euphorie/client/browser/templates/webhelpers.pt:229]: Is this normal?

Comment (reinhardt) [reply src/euphorie/client/browser/templates/webhelpers.pt:229]: IIUC it will be dealt with in https://github.com/syslabcom/scrum/issues/707

**Comment by reinhardt** [comment]
> Note to self: Check with vanilla euphorie and osha oira

Will be done in syslabcom/scrum#708

**Pull request merged**